crates/bevy_ecs/src/query/access.rs

```diff
@@ -5,7 +5,7 @@ use std::marker::PhantomData;
 #[derive(Debug, Eq, PartialEq, Clone)]
 pub struct Access<T: SparseSetIndex> {
     reads_all: bool,
-    reads_and_writes: FixedBitSet,
+    reads: FixedBitSet,
```

Comment:
The old name was more descriptive IMO.

Reply:
Haha I don't have a strong opinion here. You're going to need to hash it out with @TheRawMeatball.

Reply:
I don't feel that strongly about it either way, though I'd appreciate a comment describing what it represents if it gets changed back